src/canopy-webpack-config-esm.mjs

```diff
@@ -0,0 +1,210 @@
+import path from "path";
+import { CleanWebpackPlugin } from "clean-webpack-plugin";
+import { BundleAnalyzerPlugin } from "webpack-bundle-analyzer";
+import merge from "webpack-merge";
+import fs from "fs";
+import { homedir } from "os";
+
+let isDevServer = false;
+if (process.argv.some((arg) => arg.includes("serve"))) {
+  isDevServer = true;
+}
+
+const hostIndex = process.argv.findIndex((arg) => arg === "--host");
+const host =
+  hostIndex >= 0 && process.argv[hostIndex + 1]
+    ? process.argv[hostIndex + 1]
+    : "0.0.0.0";
+
+const portIndex = process.argv.findIndex((arg) => arg === "--port");
+const port =
+  portIndex >= 0 && process.argv[portIndex + 1]
+    ? process.argv[portIndex + 1]
+    : "8080";
+
+// --- Externals matching helpers ---
+const externalPatterns = [
+  /^canopy-sofe-extensions$/,
+  /^lodash\/?.*$/,
+  /^moment$/,
+  /^luxon$/,
+  /^online-listener$/,
+  /^prop-types$/,
+  /^react-dom$/,
+  /^react-dom\/client$/,
+  /^react-dom\/server$/,
+  /^react\/lib.*/,
+  /^react$/,
+  /^react\/jsx-runtime$/,
+  /^rx$/,
+  /^rxjs\/?.*$/,
+  /^single-spa-canopy$/,
+  /^single-spa-react$/,
+  /^single-spa$/,
+  /^react-hook-form$/,
+  /^@datadog\/browser-rum$/,
```

Comment:
I dont think we are using DD rum are we? or maybe backend is actually using the traces, just curious if we need to add this, since its not duplicated anywhere, and not on the FE node server, we probably dont want to take this out of our bundles.

Reply:
thought it was a copy/paste - good eye